app/modules/account.js

```diff
@@ -18,8 +19,15 @@ export function logout(){
   }
 };
 
+export function showKey(isToggled) {
```

Comment:
Great! In my new commit I switched it to a `React.Component` with some appropriate React-specific version cleanup and compatibility with Redux. I appreciate the help.